BunqSdk/Model/Core/BunqModel.cs

```diff
@@ -146,5 +146,7 @@ public override string ToString()
         {
             return BunqJsonConvert.SerializeObject(this);
         }
+
+        public abstract bool IsAllFieldNull();
```

Comment:
Is it correct `BunqModel` does not implement `IAnchorObjectInterface`, but still defines a `IsAllFieldNull()` method? Consider extracting this method to its own interface, and implement that.